portfolio/src/main/webapp/script.js

```diff
@@ -0,0 +1,23 @@
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     https://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+
+fetch('/data').then(response => response.json()).then((comment) => {
+	let doc = document.getElementById('comment-container');
+	for(x in comment){
```

Comment:
JavaScript's `for...in` operator is kind of weird; it doesn't guarantee the order of iteration, so you might find that comments are displayed in an unusual or inconsistent order. https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Statements/for...in

Instead, we can use `comment.forEach()` or `for (x of comment)` to iterate over this array.